README.md

```diff
@@ -21,7 +21,7 @@ This information reflects the head of this branch.
 
 | Compatible with CSI Version                                                                | Container Image             | Min K8s Version | Recommended K8s Version |
 | ------------------------------------------------------------------------------------------ | ----------------------------| --------------- | ----------------------- |
-| [CSI Spec v1.2.0](https://github.com/container-storage-interface/spec/releases/tag/v1.2.0) | quay.io/k8scsi/csi-attacher | 1.15            | 1.17                    |
+| [CSI Spec v1.2.0](https://github.com/container-storage-interface/spec/releases/tag/v1.2.0) | quay.io/k8scsi/csi-attacher | 1.15            | 1.18                    |
```

Comment:
all our release notes and documentation say min is 1.14. Do you know why it's 1.15 here?

Reply:
It was added in https://github.com/kubernetes-csi/external-attacher/pull/155, without any obvious reason (at least I don't see it in changelog).

Reply:
Fixed to 1.14